Tests/ASDisplayNodeTests.mm

```diff
@@ -2149,12 +2149,9 @@ - (void)testThatOnDidLoadThrowsIfCalledOnLoadedOffMain
 {
   ASTestDisplayNode *node = [[ASTestDisplayNode alloc] init];
   [node view];
-  dispatch_semaphore_t sem = dispatch_semaphore_create(0);
-  [NSThread detachNewThreadWithBlock:^{
```

Comment:
This test case was failing on iOS 9 because `detachNewThreadWithBlock` is available since iOS 10.